src/mod_zone_difficulty_handler.cpp

```diff
@@ -49,6 +49,7 @@ void ZoneDifficulty::LoadMapDifficultySettings()
     NerfInfo[DUEL_INDEX][0].SpellDamageBuffPct = 1;
 
     // Heroic Quest -> MapId Translation
+    HeroicTBCQuestMapList[47] = 90135; // Razorfen Kraul
```

Comment:
This is custom content unrelated to this module. It can not be merged like that.
A solution could be moving this to conf/db.